enhancements/console/guided-tours-crd.yaml

```diff
@@ -0,0 +1,163 @@
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: GuidedTourSpec is the desired guided tour configuration.
+          type: object
+          required:
+          - version
+          - displayName
+          - introduction
+          - tasks
+          properties:
+            version:
+              description: version definest the release version of the guided
+                tour. Each Guided Tour needs to be versioned, to accuratelly
+                reflect the workflow in particular release of OpenShift's web
+                console.
+              type: string
+              pattern: ^release-4.\d+$
+            displayName:
+              description: displayName is the display name of the Guided Tour
+              type: string
+            followingGuidedTour:
+              description: followingGuidedTour determines which Guided Tour should
+                be suggested to take by the user after finishing the current Guided Tour
+              type: string
+            duration:
+              description: duration informs user on aproximately how long the
+                Guided Tour should take to finish (optional)
+              type: integer
+            description:
+              description: description is the description of the Guided Tour (optional)
+              type: string
+            prerequisites:
+              description: prerequisites describes what are the prerequisites
+                for the Guided Tour (optional)
+              type: string
+            introduction:
+              description: introduction describes the purpose and usability of
+                the Guided Tour
+              type: object
+              required:
+              - about
+              properties:
+                about:
+                  description: about introduces the Guided Tour purpose
+                  type: string
+                usability:
+                  description: usability is an array of namespace names in which
+                    the dashboard link should appear (optional)
+                  type: array
+                  items:
+                    type: object
+                    required:
+                    - title
+                    - description
+                    properties:
+                      title:
+                        description: title contains a title for a particular usability feature
+                        type: string
+                      description:
+                        description: description describes a particular usability feature
+                          that is offered to the user by completing the Guided Tour
+                        type: string
+            tasks:
+              type: array
```

Comment:
I am not sure why `tasks` needs to be this structured. Each task could essentially be a single markdown. There is no automated checks, execution of tasks, progress monitoring. I can not think of a reason why we need this structure other than some marginal control over rendering. 

It would have been great if we could have more improved capabilities like ability to link to parts of console, copy commands. Define criteria to follow progress.

Reply:
Yeah makes sense to drop the structuring and keep it simple